src/modules/postScore.js

```diff
@@ -0,0 +1,17 @@
+import displayDOM from './displayDOM.js';
+import getScore from './getScores.js';
+
+const postScore = async (name, score) => {
+  await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/HX42ejstWWGWDKZzkjqD/scores/', {
+    method: 'POST',
+    headers: { 'Content-type': 'application/json; charset=UTF-8' },
+    body: JSON.stringify({ user: name, score }),
+  })
+    .then((response) => response.json())
+    .then(() => getScore())
+    .then(() => displayDOM());
+
+  return getScore();
+};
```

Comment:
- [ ] The same applies here, please consider using `async` and `await`. 🤓 🔧 

- [ ] Also, I think that you shouldn't return again the `getScore()` function, this will fire again the fetch request but you are not using it so you're ending up with two calls to the API. 🤓 🔧

Reply:
I have fixed both issues.
Thank You.